### DISCUSSION TITLE
Using pydpf to postprocess LS-DYNA results

### DISCUSSION THREAD

**Discussion by mhoeijm** [ansys/pydpf-core#495] [Q&A] [answered]
Do we already have the option to read, visualize, and post-process LS-DYNA d3plot results with pydpf? And if so, are there any examples available?

**Comment by pmaroneh** [comment, marked as answer]
Hi @ansmhoeij , AFAIK postprocessing of LS-Dyna files is available from 2022R2. It supports d3plot, binout and moddynout files. I don't think there are examples exposed yet but the syntax is very similar to the one used for postprocessing of MAPDL result files. See screenshot from a demo below.
![image](https://user-images.githubusercontent.com/76440649/191689458-68002eb5-df44-44e9-af3e-642a67a6c069.png)

**Reply by mhoeijm** [reply]
@pmaroneh thank you for the quick reply. I will have a look and let you know if I run into any problems.

**Comment by aravantaa** [comment]
@ansmhoeij
Make sure to try out LS-Reader too, it has been developed by the LS-Prepost team:
https://pypi.org/project/lsreader/
It has many capabilities and is very fast.
